src/urbit/mar/chess/game.hoon

```diff
@@ -9,15 +10,15 @@
   |%
   ++  noun  game
   ++  json
-    =+  game
     %-  pairs:enjs
-    :~  ['gameID' [%s (scot %da game-id)]]
-        ['event' [%s event]]
-        ['site' [%s site]]
-        ['round' [%s (round-string:chess round)]]
-        ['white' [%s (player-string:chess white)]]
-        ['black' [%s (player-string:chess black)]]
-        ['result' [%s ?~(result '' u.result)]]
+    :~  ['gameID' [%s (scot %da game-id.game)]]
+        ['event' [%s event.game]]
+        ['site' [%s site.game]]
+        ['round' [%s (round-string:chess round.game)]]
+        ['white' [%s (player-string:chess white.game)]]
+        ['black' [%s (player-string:chess black.game)]]
+        ['result' [%s ?~(result.game '' u.result.game)]]
+        ['moves' [%a ~]]
```

Comment:
Is moves necessary here? I thought that the frontend would just cast the typeless object returned by `urbit/http-api` as a `GameInfo` and ignore missing fields or auto-populate them as `null`.

Reply:
I assumed that was the case, but I included `moves` thinking it was just stylistically better to explicitly send `~` than implicitly send nothing. Tested just now and turns out we do need it.

Reply:
Yeah that sounds right.